content/v0.75.2.md

```diff
@@ -0,0 +1,129 @@
++++
+title = "0.75.2 Maintenance Release Notes"
+date = 2020-10-25
++++
+
+Installation instructions:
+
+- [Linux](/downloads/linux/) (or [Steam on Linux](/downloads/linux#steam))
+- [Windows](/downloads/windows/)
+- [macOS](/downloads/macos/)
+
+## Table of Contents
+
+- [1. Summary](#1-summary)
+- [2. New Features in Detail](#2-new-features-in-detail)
+  - [2.1. Select Monochrome Palette Colour](#2-1-select-monochrome-palette-colour)
+  - [2.2. Raw Mouse Input](#2-2-raw-mouse-input)
+- [3. Bugfixes](#3-bugfixes)
+- [4. Notes for Packagers](#4-notes-for-packagers)
+- [5. Contributors](#4-contributors)
+- [6. Changelog](#5-changelog)
+
+## 1. Summary
+
+This is a maintenance release from 0.75.x stable branch - it includes a few
+changes and bugfixes backported from upcoming 0.76.x version.
+
+## 2. New Features in Detail
+
+### 2.1. Select Monochrome Palette Colour
+
+To be used with `machine = hercules` or `machine = cga_mono`. Users can
+now change the default monochrome palette colour (white, paperwhite, green,
+or amber) in a configuration file:
```

Comment:
The color palette for monochrome machines, such as `machine = hercules` or `machine = cga_mono`, can now to be set to either white, paperwhite, green, or amber in the configuration file, as follows:

Reply:
I think `machine` setting name is confusing to users (not everyone will know Hercules with graphics card, the setting suggests it is a type of computer), so how slightly different wording:

"When emulating monochrome displays (via `machine = hercules` or `machine = cga_mono`), the default colour palette can now be set to either `white`, `paperwhite`, `green`, or `amber` in the configuration file, as follows:

Reply:
Even better; thanks for the refinement @dreamer.